test/Chirp.TestE2E/E2ETests.cs

```diff
@@ -695,6 +695,93 @@ public async Task TestShareCheepsVisibilityPrivateTimeline()
         // Clean up
         await DeleteUser();   
     }
+    [Test]
+    [Category("End2End")]
+    public async Task CheepingCheeps()
+    {
+        await RegisterUser();
+        await LoginUser();
+        
```

Comment:
Make sure that the test goes to the public timeline before testing it. Right now it does, but in the future we _might_ change it up. To avoid this to unnecessary fail, you can add this line to go to the public timeline: `await _page!.GotoAsync($"{AppUrl}");`